src/common/hooks/useFacets.js

```diff
@@ -0,0 +1,447 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { FormattedMessage, useIntl } from 'react-intl';
+import _ from 'lodash';
+import { languageOptionsES } from '../../components/InstanceFilters/languages';
```

Comment:
Please add empty line here to separate global deps from stripes and local

Reply:
Thanks, fixed.